tensorflow/python/ops/array_ops.py

```diff
@@ -575,6 +579,17 @@ def _slice_helper(tensor, slice_spec, var=None):
     TypeError: If the slice indices aren't int, slice, ellipsis,
       tf.newaxis or scalar int32/int64 tensors.
   """
+  # Musikisomorphie: integrate the boolean_mask into NumPy-like operation
```

Comment:
I think you can remove this comment (you also don't need to sign your comments in the tf codebase)